documentation/modules/ROOT/pages/02_basics.adoc

```diff
@@ -101,15 +101,15 @@ Quarkus CLI::
 quarkus test
 ----
 
-NOTE: `quarkus test` starts tests in continuous testing mode (more about this below) so to return to the command line after the tests ran you will need to press (q)
+NOTE: `quarkus test` starts tests in continuous testing mode (more about this below) so to return to the command line after the tests ran you will need to press kbd:[q].
```

Comment:
This needs to be checked in your environment but it should be supported by Asciidoc: https://docs.asciidoctor.org/asciidoc/latest/macros/keyboard-macro/

Reply:
let's try it :D